src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/AddUsingsCodeActionProviderFactory.cs

```diff
@@ -12,6 +14,8 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions
 {
     internal static class AddUsingsCodeActionProviderFactory
     {
+        internal static readonly Regex AddUsingVSCodeAction = new Regex("^using (.+);$", RegexOptions.Compiled, TimeSpan.FromSeconds(1));
```

Comment:
Oh interesting I didn't realize `.` captured whitespace also so this does work for `static`

Reply:
Yep!